solidity/contracts/bridge/MovingFunds.sol

```diff
@@ -416,6 +422,11 @@ library MovingFunds {
                 uint32(block.timestamp),
                 0
             );
+            // We added a new moved funds sweep request for the target wallet
+            // so we must increment their request counter.
+            self
+                .registeredWallets[targetWalletPubKeyHash]
+                .pendingMovedFundsSweepRequestsCount++;
```

Comment:
Not in this PR but I wonder if we shouldn't emit an event in this line so the target wallet could realize they have a transaction to sweep.

Right now it's possible but a bit complicated:
- Wallet needs to monitor all `MovingFundsCommitmentSubmitted` in the network.
- Once `MovingFundsCommitmentSubmitted` is emitted, the wallet needs to check if it is on the list of `targetWallets`.
- If wallet is on the list, it needs to monitor `MovingFundsCompleted` for the given source `walletPubKeyHash`

If we emitted an event here, the wallet would only care about it. I guess the decision factor should be gas costs.